src/AppBar/AppBar.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactNode, FC, useContext } from 'react';
+import { cn } from 'reablocks';
+import { ChatContext } from '../ChatContext';
+
+export interface AppBarProps {
+  /**
+   * Content to display in the left section of the header
+   */
+  contentLeft?: ReactNode;
+
+  /**
+   * Content to display in the center section of the header
+   */
+  contentCenter?: ReactNode;
+
+  /**
+   * Content to display in the right section of the header
+   */
+  contentRight?: ReactNode;
+}
+
+export const AppBar: FC<AppBarProps> = ({
+  contentLeft,
+  contentCenter,
+  contentRight
+}) => {
+  const { theme } = useContext(ChatContext);
+
+  return (
+    <div className={cn(theme?.header?.base)}>
+      <div className="flex items-center justify-between w-full">
```

Comment:
probably need to move these classes to the theme too, or maybe don't need it since `content` will allow them to take care of whatever styling they want in here